src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -2302,7 +2303,8 @@ because xbuild doesn't support framework reference assemblies.
     LibraryProjectJars="@(ExtractedJarImports)"
     TlsProvider="$(AndroidTlsProvider)"
     UncompressedFileExtensions="$(AndroidStoreUncompressedFileExtensions)"
-	InterpreterEnabled="$(_AndroidUseInterpreter)">
+	InterpreterEnabled="$(_AndroidUseInterpreter)"
+    ProjectFullPath="$(MSBuildProjectFullPath)">
```

Comment:
Fast Deployment has a different call for `<BuildApk/>` so this change will be needed here, too: 

https://github.com/xamarin/monodroid/blob/e795bcef375bc77da0bbe995fc9e57560ed5ddc0/tools/msbuild/Xamarin.Android.Common.Debugging.targets#L271-L297

You might also add the missing `InterpreterEnabled` value, too.